apps/SageAccountWeb/src/AppInitializer.tsx

```diff
@@ -53,6 +54,9 @@ function AppInitializer(props: { children?: React.ReactNode }) {
   }, [])
 
   const sourceApp = useSourceApp(appId)
+  if (sourceApp?.friendlyName) {
+    setCurrentAppInfo(sourceApp.friendlyName)
+  }
```

Comment:
Update OneSage local storage with current app

Reply:
I originally tried `useEffect` here, but LoginMethodButton looked up `currentSourceAppName` in localStorage before the effect ran. Since `currentSourceAppName` was only updated in localStorage, rather than in state, the LoginMethodButton wasn't notified that `currentSourceAppName` had been updated and so wasn't re-rendered. 

Based on [your comment](https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/369#discussion_r1265541681), I've refactored to use `useLocalStorageValue` from `@react-hookz/web`, so that `currentSourceAppName` is stored in state and can be updated via an effect.